onebusaway-android/src/main/java/org/onebusaway/android/tripservice/TripService.java

```diff
@@ -125,6 +129,20 @@ public void onStart(Intent intent, int startId) {
 
     @Override
     public int onStartCommand(Intent intent, int flags, int startId) {
+        Log.d(TAG, "Trip service started");
+        //create notification regarding foreground
+        Intent notificationIntent = new Intent(this, TripService.class);
+
+        PendingIntent pendingIntent = PendingIntent.getActivity(this, 0,
+                notificationIntent, 0);
+
+        Notification notification = new NotificationCompat.Builder(getApplicationContext(), Application.CHANNEL_TRIP_PLAN_UPDATES_ID)
+                .setSmallIcon(R.drawable.ic_stat_notification)
+                .setContentTitle("Service running in foreground")
+                .setContentText("OneBusAway will be running a service in foreground")
+                .setContentIntent(pendingIntent).build();
+
+        startForeground(1001,notification);
```

Comment:
Could you change the ID here to a larger number like `1000000`, and refactor it to a `public static final` member constant in the class?  We don't want this to collide with other notification IDs we use in OBA.  A large number seems the best shot for this to me, but let me know if you have any other ideas.

Reply:
Thank you for reviewing the PR. I will add the changes and test if it works as expected.

Reply:
@barbeau I have updated the PR. Can you please review it and let me know if any further changes are required to be made?

Reply:
@barbeau Thank you for updating the PR. I have tested the changes and everything is working as expected.